x/ecocredit/client/testsuite/tx.go

```diff
@@ -331,6 +331,88 @@ func (s *IntegrationTestSuite) TestTxCreateBatchCmd() {
 func (s *IntegrationTestSuite) TestTxSendCmd() {
```

Comment:
question: how do i run these tests? `go test` doesn't seem to find them

Reply:
`make test` will run all tests including integration tests

Reply:
`make test` will run all unit and integration tests

Reply:
you can run _only_ the receiver suite by running the `TestEcocreditIntegration` in `app/testsuite/modules_test.go`